R/visualizeDependencies.R

```diff
@@ -2,25 +2,26 @@
 #'
 #' Creates a graphical visualization of dependencies between functions in the mr-universe.
 #'
-#' @param functions function or vector of functions to be analyzed
+#' @param ... function or vector of functions to be analyzed
```

Comment:
this can also be multiple arguments now, right? Then it should be "functions or vector of functions to be analyzed" I would say. Or "function(s)"